_includes/pbc/all/install-features/202307.0/install-the-file-manager-feature.md

```diff
@@ -149,6 +149,13 @@ Ensure the following transfers have been created:
 
 3. Add the following configuration to your project:
 
+{% info_block warningBox "Note" %}
+
+The precedence of the web server's maximum file size configuration is consistently prior to the application configuration and can be adjusted using `deploy.*.yml`.
```

Comment:
@romanhavrylko Please rephrase this sentence, difficult to understand meaning

Reply:
@AlexSlawinski, updated, please take a look